bedrock/settings/static_media.py

```diff
@@ -1165,6 +1165,14 @@
         ),
         'output_filename': 'js/firefox_developer_firstrun-bundle.js',
     },
+    # will be needed when videos are ready (shortly after 6/2?)
```

Comment:
Ah! Good catch. It will be needed until the `dev-edition-spring-campaign` waffle switch is activated.